src/Storage.js

```diff
@@ -22,7 +14,7 @@ class Storage {
 
     store(streamMessage) {
         const insertStatement = 'INSERT INTO stream_data (id, partition, ts, sequence_no, publisher_id, payload) VALUES (?, ?, ?, ?, ?, ?)'
-        return callbackToPromise(this.execute, insertStatement, [
+        return this.execute(insertStatement, [
```

Comment:
Now with the `callbackToPromise` hack removed, lines 10-12 with the `bind` hacks can maybe also be removed?

The expected thing would be to just have `this.cassandraClient = cassandraClient` in the constructor and then elsewhere call `this.cassandraClient.execute(...)` instead of `this.execute(...)`?